03-Node-npm-modules/ver.0.2/downloableMovie v0.2.js

```diff
@@ -0,0 +1,107 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+
+var Movie = function (optParams) {
```

Comment:
As we discuss the other day, I'd prefer to use mutiple parameters instead of an object